src/%IPM/Storage/ResourceReference.cls

```diff
@@ -177,6 +176,11 @@ ClassMethod GetChildren(pResourceName As %String, pModuleName As %String, pCheck
 					Set pResourceArray(tFileName,"Generated") = 1
 				}
 			}
+			If (tHidden = $Char(0)) {
```

Comment:
This should happen outside the catch block. If `tHidden` is not defined, nothing to do. Otherwise have this block